conans/server/service/common/search.py

```diff
@@ -33,10 +32,8 @@ def _get_local_infos_min(server_store, ref, look_in_all_rrevs):
                 info_path = os.path.join(server_store.package(pref), CONANINFO)
                 if not os.path.exists(info_path):
                     raise NotFoundException("")
-                conan_info_content = load(info_path)
-                info = ConanInfo.loads(conan_info_content)
-                conan_vars_info = info.serialize_min()
-                result[package_id] = conan_vars_info
+                content = load(info_path)
+                result[package_id] = {"content": content}
```

Comment:
I am rethinking this, maybe we need to keep the legacy "settings" and other fields if possible, it looks like that Artifactory will not be able to break this way, so maybe try to be consistent.

Reply:
I don't get what you mean

Reply:
Artifactory cannot return only the ``{"content": `` field immediately, because it will break all the older Conan clients, and migrations will take a while. So Artifactory will need to return both ``"content"`` key, plus the legacy "settings", "options", etc, to avoid breaking, even if in some cases, like when uploading a new modern 2.0 conaninfo.txt, it doesn't get those fields, and they are blank, but they should still be returned to the client side with ``{"settings": ..., "options": ...., "content": ....}`` to avoid old clients crashing.